src/redivis/common/list_rows.py

```diff
@@ -92,10 +93,30 @@ def list_rows(
         progressbar = tqdm(total=max_results, leave=False)
 
     if output_type == 'arrow_iterator':
-        return RedivisArrowIterator(streams=read_session["streams"], mapped_variables=mapped_variables, progressbar=progressbar, coerce_schema=coerce_schema )
+        return RedivisArrowIterator(
+            streams=read_session["streams"],
+            mapped_variables=mapped_variables,
+            progressbar=progressbar,
+            coerce_schema=coerce_schema
+        )
+
+    if folder is None:
```

Comment:
Per comment above, we'll want to refactor this logic.

However, I was intending to look at an env variable, `REDIVIS_TMPDIR`, as part of this fix. So basically, something like:
```py

     folder = (
            pathlib.Path("/")
                .joinpath(
                    os.getenv("REDIVIS_TMPDIR") or tempfile.gettempdir(),
                    "redivis",
                    "tables",
                    f"{uuid.uuid4()}"
    )